src/index.scss

```diff
@@ -0,0 +1 @@
+@use "./base";
```

Comment:
Does this mean we aren't including any component CSS yet?

Reply:
That was a mistake, I fixed it in https://github.com/cloudfour/cloudfour.com-patterns/pull/569/commits/78210ab34358d7f7e002086460254cb99839ed51